internal/ingress/ingress.go

```diff
@@ -153,16 +150,46 @@ func EnumerateIngresses(ctx context.Context, k8sconfig *rest.Config, authType me
 
 				rules := []*methodk8s.Rule{}
 				for _, rule := range ingress.Spec.Rules {
+					baseHost := rule.Host
+					service, err := clientset.CoreV1().Services(ingress.GetNamespace()).Get(ctx, rule.HTTP.Paths[0].Backend.Service.Name, metav1.GetOptions{})
+					if err == nil && service.Spec.Ports != nil && len(service.Spec.Ports) > 0 {
+						port := fmt.Sprintf(":%d", service.Spec.Ports[0].Port)
+						baseHost += port
+					}
+
 					for _, path := range rule.HTTP.Paths {
 						ruleInfo := methodk8s.Rule{
 							Path:        path.Path,
-							Base:        rule.Host,
```

Comment:
@payoub125 , does the rule ever have a host? We want to use this base host for every possible path?